server/src/main/java/com/saralein/server/request/parser/RangeParser.java

```diff
@@ -0,0 +1,61 @@
+package com.saralein.server.request.parser;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RangeParser {
+    Map<String, Integer> parse(String range) {
```

Comment:
Currently using a map for range.  Either empty, with start, with end, or with both start and end.